psaw/PushshiftAPI.py

```diff
@@ -114,10 +114,12 @@ def _query(self, kind, stop_condition=lambda x: False, **kwargs):
 
     def _comments(self, kind, **kwargs):
         base36_id = kwargs.get('base36_id', None)

```

Comment:
`base36_id` is unidiomatic. If the method is `get comments_of_post`, the argument should be `submission_id`, since that's the only id it will accept. Otherwise, if we were going to be generic, it should  probably be one of `name`, `id`, or `thing_id`, since that's how IDs are referred to in reddit docs. I don't think I've ever seen anything referred to as a "base36_id".